### DISCUSSION TITLE
Navigating to first commit of repository

### DISCUSSION THREAD

**Discussion by ManojkumarBairi** [community/community#42566] [Repositories]
### Select Topic Area

Product Feedback

### Body

In github web, If a repository has thousands of commits,then to navigate to the initial commits with Newer/Older button is too painfull.Can you please check if there is any other way if not can you please provide proper pagination.

**Comment by jfullstackdev** [comment]
as a workaround, you can use GitHub Desktop, 

it's as simple as scrolling down at the History tab
